HOWTO-port-an-exercise.md

```diff
@@ -15,34 +15,38 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
     - add the labels: `x:action/create`, `x:module/practice-exercise`,
       `x:status/claimed`, `x:type/content`
     - assign it to yourself.
-1. copy the exercises/practice/foo-bar directory from the bash repo.
-1. in the exercises/practice/foo-bar directory, **delete** `foo_bar.sh`, and
+1. set `exercise=foo-bar`
+1. copy the exercises/practice/$exercise directory from the bash repo.
+1. in the exercises/practice/$exercise directory, **delete** `${exercise/-/_}.sh`, and
    `.meta/{test.toml,version,example.sh}`
-1. rename `foo_bar.bats` to `test-foo-bar.bats`
+1. rename `${exercise/-/_}.bats` to `test-$exercise.bats`
     - note the underscore changed: the AWK track uses kebab case
-1. edit `.meta/config.js`
+1. edit `.meta/config.json`
     - fix the filenames from .sh to .awk, and the test file
     - set yourself as the author, and empty the contributors list.
-1. edit `test-foo-bar.awk` so the tests are AWK specific. 
+1. edit `test-$exercise.awk` so the tests are AWK specific. 
```

Comment:
We both missed this: this should be `test-$exercise.bats`

Reply:
Done